index.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML + basic JavaScript Capstone</title>
+
+    <!-- stylesheet -->
+    <link rel="stylesheet" href="styles.css">
+
+    <!-- font family -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap"
+      rel="stylesheet">
+
+    <!-- boostrap -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous">
+    <!-- icons -->
+    <link rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.3/font/bootstrap-icons.css">
+  </head>
+
+
+  <body>
+    <header class="header">
+      <div class="navbar navbar-expand-lg navbar-dark bg-dark navbar-social">
+        <div class="container-fluid ">
+          <div class="collapse navbar-collapse justify-content-end
+            navbar-nav-social" id="navbarNav"><div>
+              <ul class="navbar-nav nav-bar-items-social">
+
+                <li><a href=""><img src="images/facebook.png" alt="facebook
+                      logo"></a></li></div>
+              <li><a href=""><img src="images/twitter.png" alt="twitter logo"></a></li>
```

Comment:
- [ ] Great job working on the social media icons, however, they are not displaying due to broken links. This is because your links start with a small letter `i` for the images folder when your actual folder has the capital letter `I`. Kindly fix this

Your design
![image](https://user-images.githubusercontent.com/81932478/233684259-c19c4ad6-9dab-4729-9d10-4baa6ea2f96c.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/233684463-6406b0ce-3443-40fe-8abf-05686e84f72e.png)